src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -45,7 +45,7 @@
     @SqlQuery("SELECT c.consentId, cs.dataSetId, ds.name, ds.objectId " +
             "FROM consents c INNER JOIN consentassociations cs ON c.consentId = cs.consentId " +
             "INNER JOIN dataset ds on cs.dataSetId = ds.dataSetId "+
-            "WHERE cs.dataSetId IN (<datasetId>)")
+            "WHERE cs.dataSetId::text IN (<datasetId>)")
```

Comment:
This is unintuitive and took a good amount of time to debug. JDBI is quoting the integer values in `<datasetId>` in this query which fails against the `bigint` `datasetid` column. This doesn't seem to be a problem in other cases but is in this one.